backend/src/main/java/de/neuefische/backend/model/TodoElement.java

```diff
@@ -3,12 +3,16 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import lombok.With;
 
 @Data
+@With
```

Comment:
nice. escpecially when the attributes are set to final. so there are immutable. then the wither spread its wing to the fullest. cause its instantiate a new Object with all the other values